example/integration_test/android_test.dart

```diff
@@ -10,6 +10,7 @@ void main() {
     'intKey': 12,
     'boolKey': true,
     'floatingNumberKey': 12.1,
+    'longKey': 8640000000000000,
```

Comment:
This should rather be `double.maxFinite`

Reply:
What difference does this make to the double test case?